silverorange/CombineFirebase
# Add support for Firebase 11

Updates Package.swift to target Firebase 11.2.0. It also updates the package code to remove dependency on `FirebaseFirestoreSwift`, which has been merged into `FirebaseFirestore`. See the [Firebase changelog](https://firebase.google.com/support/release-notes/ios#11.2.0) for details on this change.

### To Test
1. Check out this PR
2. Open it in Xcode
3. With `CombineFirebase-Package` selected as the build target, compile it.
4. It should finish without issue.

[ch63843]
